jans-cedarling/cedarling_opa/convert.py

```
@@ -0,0 +1,226 @@
+            filepath = dir_path / object
+            with open(filepath, "r", encoding="utf-8") as f:
+                try:
+                    content = json.load(f)
+                    key = object.stem
+                    aggregated[key] = content
+                except json.JSONDecodeError:
+                    print(f"Skipping file {filepath}")
+    return aggregated
+
+
+def convert_cjar_to_opa(
```

Ruff (0.15.21)

[warning] Too many branches (20 > 12)

(PLR0912)